src/clr-addons/history/history-model.interface.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2018-2020 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export interface ClrHistoryModel {
+  /**
+   * The username
+   */
+  username: string;
+
+  /**
+   * The tenant id
+   */
+  tenantid?: string;
```

Comment:
hmm just thought that "tenantId" and "applicationName" are very opinionated for a general component.
do you think we can refactor that to a context array?
and also include the existing context variable into it, as it seems it's just there to be able to add the same page multiple times with different path variables